staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -217,19 +218,33 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "code", "rejected"})
 
+	webhookSLO := metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_slo_duration_seconds",
+			Help:           "Admission webhook latency histogram in seconds, identified by name and broken out for each operation and for a configurable set of API resources.",
+			Buckets:        []float64{0.005, 0.025, 0.1, 0.5, 1.0, 2.5},
```

Comment:
since webhook timeout is by default 10 seconds, we should probably include that as a bucket otherwise charts are going to top out at 2.5 seconds.

Reply:
Done.

Reply:
I switched to using the [default buckets](https://github.com/kubernetes/kubernetes/pull/108468#discussion_r824481548).